.github/workflows/chromatic.yml

```
@@ -7,6 +7,7 @@ on:
     types: [opened, labeled, synchronize]
 jobs:
   run:
+    if: !contains(github.event.pull_request.labels.*.name , 'skip visual snapshots')
```

Nitpick: remove the space before the comma.

```suggestion
    if: !contains(github.event.pull_request.labels.*.name, 'skip visual snapshots')
```